src/downloadImages.js

```diff
@@ -0,0 +1,27 @@
+import cowSwapList from './public/CowSwap.json' assert { type: "json" }
+import fetch from 'node-fetch';
+import {createWriteStream} from 'fs'
+import path, {dirname} from 'path'
+import { fileURLToPath } from 'url';
+
+
+
+async function main() {
+  for(const token of cowSwapList.tokens) {
+    const { symbol, name, address, chainId, logoURI } = token
+    const filePath = `public/images/${chainId}/${address}.png`
+
+    console.log(`Download image for ${symbol} (${name})
```

Comment:
Maybe only fetch if the file doesn't yet exist. If we want to change the image, feels reasonable to delete the old one first?

Reply:
sure, done. This is not meant to be executed a lot after we define a good automatic process to add tokens.

here is a test where I just deleted 3 images to download again
<img width="929" alt="image" src="https://user-images.githubusercontent.com/2352112/206586757-c5343004-9793-49ff-a8bd-910e55a46a1e.png">